core/src/main/java/org/apache/iceberg/MetricsConfig.java

```diff
@@ -77,6 +79,14 @@ public static MetricsConfig getDefault() {
   }
 
   public static MetricsConfig fromProperties(Map<String, String> props) {
+    return from(props, null);
+  }
+
+  public static MetricsConfig forTable(Table table) {
```

Comment:
Should we also use this method at `org.apache.iceberg.parquet.Parquet.WriteBuilder#forTable`, since we have replaced other usages in Parquet.java? https://github.com/apache/iceberg/pull/2240/files#diff-8b42edd2a56abae1dc480f36b0e8126c3742d4f41fd7db8ca0df358527940c4cR394

Reply:
Good catch, done

Reply:
Resolving as this seems to be done.